docs/source/modin/supported/index_supported.rst

```diff
@@ -125,7 +125,7 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``nunique``                 | Y                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``value_counts``            | N                               |                                  |                                                    |
+| ``value_counts``            | Y                               |                                  |                                                    |
```

Comment:
This should be P because there are cases where we raise NotImplementedError. Please document them here.